.circleci/config.yml

```diff
@@ -70,6 +70,7 @@ jobs:
           command: |-
             mkdir -p /home/node/.npm-global
             npm install
+            npm link
```

Comment:
You don't need this :)  The `npm link ../` you have below does it for you.

Reply:
Got it! lint was acting out again so I was trying different things 😞 ended up regenerating samples/package-lock.json